Base.rte/Devices/Tools/Constructor/Constructor.ini

```diff
@@ -471,38 +488,55 @@ AddActor = Actor
 	ScriptPath = Base.rte/Devices/Tools/Constructor/Cursor.lua
 	SpriteFile = ContentFile
 		FilePath = Base.rte/Null.bmp
+	FrameCount = 1
+	SpriteOffset = Vector
+		X = -1
+		Y = -1
 	Buyable = 0
 	AtomGroup = AtomGroup
-		CopyOf = Null AtomGroup
+		CopyOf = Constructor Null AtomGroup
+	DeepCheck = 0
 	Status = 0
 	Health = 100
 	Team = 0
 	Perceptiveness = 0.0
+	AimAngle = 0
 	AimDistance = 0
 	CharHeight = 99999
+	HolsterOffset = Vector
+		X = -0
+		Y = -0
 	GibImpulseLimit = 10000
 	GibWoundLimit = 10000
 
 
-///////////////////////////////////////////////////////////////////////
-// Constructor Ammo
+AddAmmo = Round
+	PresetName = Round Constructor
+	ParticleCount = 1
+	Particle = MOPixel
+		CopyOf = Null Bullet
+	Shell = None
+	FireVelocity = 100
+	ShellVelocity = 10
+	Separation = 10
 
 
 AddAmmo = Magazine
 	PresetName = Magazine Constructor
-	Mass = 5
+	Mass = 1
 	HitsMOs = 0
 	GetsHitByMOs = 0
-	CollidesWithTerrainWhenAttached = 1
+	ScriptPath = Base.rte/Devices/Tools/Constructor/ConstructorMag.lua
 	SpriteFile = ContentFile
-		FilePath = Base.rte/Devices/Tools/Constructor/ConstructorMagazine.bmp
```

Comment:
If you're no longer using this sprite, delete it.
Try to minimize unused junk laying around, otherwise base will look like it was before the refactor.